bindings/rust/bench/src/rustls.rs

```diff
@@ -72,131 +68,157 @@ impl RustlsHarness {
     }
 }
 
-impl TlsBenchHarness for RustlsHarness {
-    fn new(
+/// Clients and servers have different config types in Rustls, so wrap them in an enum
+pub enum RustlsConfig {
+    Client(Arc<ClientConfig>),
```

Comment:
This might be just my rust ignorance, but you're not opening new threads in benchmarking are you? So why do you need an Arc?

Reply:
Rustls just requires an `Arc<ClientConfig>` to make a new connection, not just a `ClientConfig`. Probably it's because Rustls tries to force you to use best practice?